codechecker_lib/analyzer.py

```diff
@@ -36,6 +39,24 @@ def prepare_actions(actions, enabled_analyzers):
     return res
 
 
+def __print_analyzer_version(context, analyzer_config_map):
+    """
+    Print the path and the version of the analyzer binary.
+    """
+    check_env = analyzer_env.get_check_env(context.path_env_extra,
+                                           context.ld_lib_path_extra)
+
+    # get the analyzer binaries from the config_map which
+    # contains only the checked and available analyzers
+    for analyzer_name, analyzer_cfg in analyzer_config_map.iteritems():
+        LOG.info("Using analyzer: ")
+        analyzer_bin = analyzer_cfg.analyzer_binary
+        print(analyzer_bin)
```

Comment:
The logging module formats the output differently which I think in this case is less readable.